services/hierarchy/src/test/kotlin/PackageServiceTest.kt

```diff
@@ -196,7 +196,7 @@ class PackageServiceTest : WordSpec() {
 
                 service.countForOrtRunIds(ortRun1Id, ortRun2Id) shouldBe 3
             }
-    }
+        }
```

Comment:
Nit: Not for this time, but maybe the next: For single-file changes you may use the name of the file as the scope of the change in the commit title, like `chore(PackageServiceTest): ...`.

Reply:
I'd rather keep the scopes limited to module names: https://github.com/eclipse-apoapsis/ort-server/blob/main/CONTRIBUTING.md#scopes